src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -158,6 +158,12 @@ Blank()
 >> DateTimeValue(ParseJSON("""abcdef"""))
 #Error(Kind=InvalidArgument)
 
+>> Text(DateTimeValue(ParseJSON("""2022-05-05""")))
+"5/5/2022 12:00 AM"
```

Comment:
This will probably fail for anyone that uses a 24h clock locale (e.g., most people in Europe). Something like below should be more "universal":

    >> Text(DateTimeValue(ParseJSON("""2022-05-05""")), "yyyy-MM-dd HH:mm:dd")
    "2022-05-05 00:00:00"

Reply:
Fixed